taskcluster/ci/signing/kind.yml

```diff
@@ -62,7 +62,7 @@ job-template:
           default: autograph_apk
     index:
         by-build-type:
-            (fennec-.+|nightly|performance-test|beta|production):
+            (fennec-.+|nightly|performance-test|beta|production|debug):
```

Comment:
What's the reason for that? They won't be signed so not really installable without manual work.